### DISCUSSION TITLE
Customizing the <title> tag from "Scalar API Reference"

### DISCUSSION THREAD

**Discussion by tmcw** [scalar/scalar#1910] [Q&A]
I'm using `@scalar/fastify-api-reference` with Fastify, and I've customized the title of my docs via the `openapi.info.title` option. I'd like to customize the title tag of the page, though, and it looks like this is hardcoded. Is it? Is it customizable? Thanks!

**Comment by hanspagel** [comment]
It should, but it isn’t! Please subscribe to this issue: https://github.com/scalar/scalar/issues/709

I tried to fix it in March, but couldn’t make it work. Probably about time to give it another try, thanks for reminding me.

BTW VAL.TOWN IS SO GREAT

**Reply by tmcw** [reply]
Thank you! Subscribed
